src/Styleguide/Components/EntityOverline.tsx

```diff
@@ -0,0 +1,78 @@
+import { Avatar, Box, Flex, Sans, Serif } from "@artsy/palette"
+import FollowArtistButton from "Components/FollowButton/FollowArtistButton"
+import React, { Component } from "react"
+// import { AvatarInitials } from "Styleguide/Components/AvatarInitials"
```

Comment:
This is moved as part of avatar now, right?

Reply:
Yup -- will remove comment